Handle suggestions on focus

nomego opened the pull request:
This enables `minLength: 0` usage and also re-displays suggestions
on blur/re-focus.

Commit:
Handle suggestions on focus

This enables `minLenght: 0` usage and also re-displays suggestions
on blur/re-focus.

jhuesos reviewed (commented):
First of all, thanks for your contribution. 

But, would you mind adding some tests?

Commit:
Fix condition to query for suggestions

With minLength 0, value will be '', which evaluates as falsy.
Explicitly check for null or undefined instead.

Signed-off-by: Patrik Kullman <patrik.kullman@neovici.se>

jhuesos commented on paper-autocomplete-suggestions.html:
```diff
@@ -521,7 +521,7 @@
         var value = event.target.value;
 
-        if (value && value.length >= this.minLength) {
+        if (value !== null && value !== undefined && value.length >= this.minLength) {
```
you can do value != null to check against both null and undefined

Commit:
Make loose null comparison to also catch undefined, instead of
explicit checks.

jhuesos commented:
Great. Many thanks I will merge this one.

Next time, please try to also write the test. Otherwise it is more work for me and it will take longer to release the fix.

jhuesos commented:
Available in `v3.2.0`

nomego commented:
Great, thanks. Yes it was on the TODO.. it's just a rather long one. :)
